lib/generate-id.js

```diff
@@ -22,11 +22,41 @@ const findId = async ( model, length ) => {
 }
 
 /**
- * Generates a line id and a shabad id, using the database.
+   * Logs the generated id to the console
+   * @param {Number} count the count of id to generate
+   * @param {String} type shabad, line
+   */
+const logGeneratedID = async function ( count, type ) {
+  let i = 0
```

Comment:
In JS, put your let before the variable in the loop. It is also preferable to use `i++` (but 90% of the time, we can avoid `for` loops altogether in favour of `.map` etc, this is one of the reasonable exceptions)

```
for ( let i = 0; i < count; i++ )
```

Reply:
If I use `i++` then eslint gives me following error ` Unary operator '++' used.eslintno-plusplus `